core/executor/src/system_contract/image_cell/abi/README.md

```diff
@@ -1,14 +1,15 @@
 # Generate `ImageCell.abi.json`
 
-`ImageCell.abi.json` is generated by [hardhat](https://hardhat.org/tutorial/writing-and-compiling-contracts).
+API is generated by [hardhat](https://hardhat.org/tutorial/writing-and-compiling-contracts).
 
 ```
 cd ../contract
+yarn
```

Comment:
I just follow the steps here 
> https://hardhat.org/tutorial/creating-a-new-hardhat-project
No need to execute `yarn`.

Reply:
`yarn` will install all dependencies. It only needs to be executed once.